src/dialog.tsx

```diff
@@ -257,6 +279,34 @@ const useNativeDialog = ({
     }, [onAcceptRef, onCancelRef, acceptText, cancelText, showCancel, message, title, renderNative]);
 };
 
+type NativeModalDialogProps = ModalDialogProps & {
+    dialogAcceptButton: string;
+    dialogCancelButton: string;
+};
+
+const NativeModalDialog = (props: NativeModalDialogProps) => {
+    const renderNative = isWebViewBridgeAvailable();
```

Comment:
why is this condition repeated now?

Reply:
true, not needed here, because will be always `true`, removed